packages/mds-compliance/api.ts

```diff
@@ -159,13 +159,20 @@ function api(app: express.Express): express.Express {
   })
 
   app.get(pathsFor('/count/:rule_id'), async (req: ComplianceApiRequest, res: ComplianceApiResponse) => {
+    // /*
+    const { timestamp: query_end_date } = {
+      ...parseRequest(req, { parser: Number }).query('timestamp')
+    }
+    //* /
```

Comment:
Unsure what these comments are for

Reply:
It renders like normal code in github, I can't tell if the block is commented out or not